include/service_wsrep.h

```diff
@@ -27,12 +27,19 @@ enum Wsrep_service_key_type {
   WSREP_SERVICE_KEY_EXCLUSIVE
 };
 
+enum enum_wsrep_version {
```

Comment:
Why do we need "enum_" in the name?

Also, if we are using the typename as a prefix, it could be an `enum class`.

Enums / enums classes also allow to specify the underlying type by writing  to `enum class wsrep_version : long`, to keep it the same as before.

If using the type as a prefix, and an enum class, I would also leave the `WSREP` 
 and `VERSION` out from the version names, as it's redudant. `wsrep_version::v1`

Reply:
Done